website/tables/azure/azure_subscription_subscription_locations.md

```diff
@@ -19,8 +19,9 @@ This table depends on [azure_subscription_subscriptions](azure_subscription_subs
 |_cq_id|UUID|
 |_cq_parent_id|UUID|
 |subscription_id|String|
+|metadata|JSON|
 |display_name|String|
 |id (PK)|String|
-|latitude|String|
```

Comment:
This is now a key under `metadata` along with `longitude` and bunch of new keys.

Reply:
Maybe we can keep these fields as aliases of the ones in the metadata JSON (using path  resolver) to keep this change backwards-compatible?

Reply:
> Maybe we can keep these fields as aliases of the ones in the metadata JSON (using path resolver) to keep this change backwards-compatible?

+1

Reply:
This looks good, but +1 for making this backwards compatible

Reply:
Hopefully fixed in https://github.com/cloudquery/cloudquery/pull/10369/commits/6fc0ad846c5694398fd6b80d11ff8d3de1177a5a